kube-runtime/src/controller/mod.rs

```diff
@@ -373,17 +384,24 @@ where
         self
     }
 
-    /// Indicate an object to watch with a custom mapper
+    /// Specify `Watched` object which `K` has a custom relation to and should be watched
+    ///
+    /// To define the `Watched` relation with `K`, you **must** define a custom relation mapper, which,
+    /// when given a `Watched` object, returns an option or iterator of relevant `ObjectRef<K>` to reconcile.
```

Comment:
Maybe we can relate this back to `owns` somehow?

Reply:
Adding: `If the relation K has to Watched is that K owns Watched, consider using Controller::owns.`